libs/GraphicECS/SFML/Components/AnimationFrequencyComponent.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type [WSL : Ubuntu]
+** File description:
+** AnimationFrequencyComponent
+*/
+
+#ifndef ANIMATIONFREQUENCYCOMPONENT_HPP_
+#define ANIMATIONFREQUENCYCOMPONENT_HPP_
+
+#include <chrono>
+#include "Component/Component.hpp"
+
+namespace graphicECS::SFML::Components
+{
+    /// @brief This component allows to know the frequency between each frame of an animation.
+    class AnimationFrequencyComponent : public ecs::Component {
```

Comment:
Name it as AnimationFrequency

Reply:
It's copplient with other component with others SFML component, so it shoold stay AnimationFrequencyComponent

Reply:
Ok, my bad, I didn't see that before.